docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -113,7 +117,10 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
         (warehouse, location),
         (warehouse, size),
         (warehouse))`.
- 
+    
+    `GROUP BY GROUPING SETS(GROUPING SETS(warehouse, product), GROUPING SETS(location, size))` is equivalent to 
+    `GROUP BY  GROUPING SETS((warehouse, product), (location, size))`.
```

Comment:
interesting, so nested grouping set is just syntax sugar for specifying a grouping set?

Reply:
how about
```
GROUPING SETS(GROUPING SETS((warehouse), (warehouse, product)))
```

Reply:
Done

Reply:
> interesting, so nested grouping set is just syntax sugar for specifying a grouping set?

It seems yes, the benefit is we can  write CUBE/ROLLUP in GROUPING SETS.
since `GROUP BY GROUPING SETS((A, B), (A), (), (A, C))` can write as `GROUPING BY(ROLLUP(A, B), (A, C))`